sdk/core/azure-core/inc/azure/core/http/http.hpp

```diff
@@ -91,22 +91,76 @@ namespace Azure { namespace Core { namespace Http {
   };
 
   /**
-   * HTTP request method.
-   *
+   * @brief The method to be performed on the resource identified by the Request.
    */
   class HttpMethod final {
   public:
     HttpMethod() = delete;
+
+    /**
+     * @brief Constructs a new HttpMethod from a given string.
+     *
+     * @note Won't check if \p value is a known HttpMethod defined as per any RFC.
+     *
+     * @param value A given string to represent the HttpMethod.
+     */
     explicit HttpMethod(std::string value) : m_value(std::move(value)) {}
+
+    /**
+     * @brief Compares two instances of `HttpMethod` for equality.
+     *
+     * @param other Some `HttpMethod` instance to compare with.
+     * @return `true` if instances are equal; otherwise, `false`.
+     */
     bool operator==(const HttpMethod& other) const { return m_value == other.m_value; }
+
+    /**
+     * @brief Compares two instances of `HttpMethod` for equality.
+     *
+     * @param other Some `HttpMethod` instance to compare with.
+     * @return `false` if instances are equal; otherwise, `true`.
+     */
     bool operator!=(const HttpMethod& other) const { return !(*this == other); }
+
+    /**
+     * @brief Returns the HttpMethod represented as a string.
+     */
     const std::string& ToString() const { return m_value; }
 
+    /**
+     * @brief The representation of a `GET` HttpMethod based on [RFC 2616]
+     * (https://datatracker.ietf.org/doc/html/rfc2616#section-9.3).
```

Comment:
Good stuff, but this RFC has been obsoleted by others, so we may want to refer to those instead:
![image](https://user-images.githubusercontent.com/6527137/119561319-dea49880-bd59-11eb-83be-4e865b4861fb.png)

So, try this?
https://datatracker.ietf.org/doc/html/rfc7231#section-4.3.1

Also, will the link render as expected in doxygen?

Reply:
![image](https://user-images.githubusercontent.com/24213737/119566165-b28c1600-bd5f-11eb-9158-490d01f3bc0e.png)

Reply:
Will update to 7231.  Thank you